packages/ui-components/src/ExpandableComponent.tsx

```diff
@@ -57,112 +57,88 @@ export interface IExpandableComponentProps {
   onMouseDown?: (e: React.MouseEvent<HTMLSpanElement, MouseEvent>) => any;
 }
 
-export interface IExpandableComponentState {
-  expanded: boolean;
-}
-
 /**
  * A React component for expandable containers.
  */
-export class ExpandableComponent extends React.Component<
-  IExpandableComponentProps,
-  IExpandableComponentState
-> {
-  constructor(props: IExpandableComponentProps) {
-    super(props);
-    this.state = { expanded: false };
-  }
+export const ExpandableComponent: React.FC<IExpandableComponentProps> = props => {
+  const [expanded, setExpandedValue] = React.useState(false);
 
-  toggleDetailsDisplay(): void {
+  const toggleDetailsDisplay = (): void => {
```

Comment:
it's normally best to start event handler names with "handle" so maybe `handleToggleDetailsDisplay`